src/components/views/auth/Welcome.tsx

```diff
@@ -59,6 +55,7 @@ export default class Welcome extends React.PureComponent<IProps> {
                         replaceMap={{
                             "$riot:ssoUrl": "#/start_sso",
                             "$riot:casUrl": "#/start_cas",
+                            "$matrixLogo": MATRIX_LOGO_HTML,
```

Comment:
I think we need to have a `[matrix]` replacer too to not break any deployments already relying on that magic string in custom welcome.html pages